app/Services/User/CreateAccount.php

```diff
@@ -57,30 +59,26 @@ public function execute(array $data): User
     {
         $this->validateRules($data);
 
-        $user = $this->createUser($data);
+        $this->createUser($data);
 
         if (! config('mail.verify') || User::count() == 1) {
             // if it's the first user, we can skip the email verification
-            $user->markEmailAsVerified();
-        } else {
-            $user->sendEmailVerificationNotification();
```

Comment:
@asbiin this is the line that I've deleted. And the app keeps sending emails... but only one, not two, so it's good.

Reply:
that's fine